src/helpers/generateField.ts

```diff
@@ -1,19 +1,23 @@
 import ts from "typescript";
 
-export const generateField = (
-  name: string,
-  type: ts.TypeNode,
-  nullable: boolean,
-  generated: boolean,
-  list: boolean
-) => {
+type GenerateFieldArgs = {
+  name: string;
+  type: ts.TypeNode;
+  nullable: boolean;
+  generated: boolean;
+  list: boolean;
+  documentation?: string;
+};
+export const generateField = (args: GenerateFieldArgs) => {
   /*
    * I'm not totally sure in which order these should be applied when it comes
    * to lists. Is the whole list nullable or is each entry in the list nullable?
    * If you run into problems here please file an issue or create a pull request
    * with a fix and some proof please. Thank you :D
    */
 
+  const { name, type, nullable, generated, list, documentation } = args;
```

Comment:
This is a nice way to clean the function signature up. Totally in favor of this 👍